src/Service/Resource.php

```diff
@@ -80,7 +80,7 @@ public function __construct($service, $serviceName, $resourceName, $resource)
    * @param $name
    * @param $arguments
    * @param $expectedClass - optional, the expected class name
-   * @return Request|$expectedClass
+   * @return $expectedClass|ResponseInterface|RequestInterface|array
```

Comment:
remove `array` from this typehint also
```suggestion
   * @return mixed|$expectedClass|ResponseInterface|RequestInterface
```

Reply:
Hey @bshaffer , seems like you've fixed that already